### ISSUE TITLE
Form field validation

### ISSUE THREAD

**Issue by szepeviktor** [pods-framework/pods#2967]
Do you plan to add any of these?

- [ ] Minimal length
- [ ] Maximal length
- [ ] Regexp to match

**Comment by Shelob9** [comment]
@szepeviktor Are you talking about in the Pods ACT editor, the Pods fields metabox for post type Pods or in front-end forms created with the `PodsForm` class?

**Comment by szepeviktor** [comment]
About the Pods fields metaboxes on the Dashboard.

**Comment by szepeviktor** [comment]
When I set up the fields on the "Manage Pods" page only the maximal length can be set.

**Comment by Shelob9** [comment]
@szepevikto Yes those would be great to have as options in the Pods editor. It would be a great PR to add to Pods 3.x

**Comment by szepeviktor** [comment]
Yes. Do you have plans for general field validation?

**Comment by szepeviktor** [comment]
Does the "Email" field get validated? (server side)

**Comment by sc0ttkclark** [comment]
We do have field-specific validation currently happening on both client-side (JS) and server-side (PHP). We have two things built-in like minimum length and regex matching but it was never hooked up because of work on other features.

**Comment by szepeviktor** [comment]
Thank you. That is enough for me.

**Comment by creabrain** [comment]
Please add this to next milestone or adjust the priority. A quick wine is to add js and php regex seperatly as string on field.
